### DISCUSSION TITLE
kpanda/user-guide/workloads/create-deployment/

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#2978] [Announcements]
# kpanda/user-guide/workloads/create-deployment/

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/kpanda/user-guide/workloads/create-deployment/

**Comment by LLLzxx** [comment]
高级配置 -- 调度策略 下的图片失效

**Comment by samzong** [comment]
镜像仓库部分配图已过时，缺少镜像仓库秘钥的介绍

**Comment by windsonsea** [comment]
Has been fixed in #4576
